subprojects/docs/src/docs/userguide/softwareModel.adoc

```diff
@@ -19,7 +19,7 @@
 [NOTE]
 ====
 
-Support for rule based configuration is currently <<feature_lifecycle,incubating>>. Please be aware that the DSL, APIs and other configuration may change in later Gradle versions.
+Rule based configuration will be deprecated. New plugins should not use this concept.
 
 ====
 
```

Comment:
⭕️ The text below still reads very much like a pitch for the new model. Maybe we can take some wording from our blog post to describe that this was an *attempt*, which didn't quite work out and that it's gonna be replaced by new concepts in the existing model?

Reply:
done